src/ashell/ide-cmd.c

```diff
@@ -0,0 +1,743 @@
+// Copyright (c) 2017-2018, Intel Corporation.
+
+// C includes
+#include <ctype.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
```

Comment:
what is is malloc.h  used for? Isn't this running on Zephyr?

Reply:
Apparently nothing, removed...